src/screens/Text.js

```diff
@@ -197,8 +197,10 @@ export default () => (
         <Property name="weight">
           <Description>Font weight</Description>
           <PropertyValue type="string">
+            <Example>"lighter"</Example>
             <Example>"normal"</Example>
             <Example>"bold"</Example>
+            <Example>"bolder"</Example>
```

Comment:
unless there was a rationale for the order, keeping aligned with [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#syntax) starting with most often/familiar then into relative to parent. (sorry, being nit-picky here)

```suggestion
            <Example>"normal"</Example>
            <Example>"bold"</Example>
            <Example>"lighter"</Example>
            <Example>"bolder"</Example>
```

Reply:
I just was putting it in the order that came to mind lol! I can change though